improver/cli/threshold.py

```diff
@@ -116,7 +116,10 @@ def process(
         ValueError: If threshold_config is used for fuzzy thresholding
```

Comment:
This is not a problem introduced in this PR, however I notice that 2 errors which can be raised in this CLI `("Cannot apply land-mask cube without in-vicinity processing"` and `"Can only collapse over a realization or a percentile coordinate."`) are missing from the docstring. Since this PR is changing one of these errors, it might be a good time to correct the doctsring to include them both.

Reply:
I've added these error messages to the Raises section.